README.md

```diff
@@ -636,6 +640,15 @@ Mandatory fields:
 * `id` - a string that identifies the notification, this id can be referenced by `vod_notification_uri`
 	using the variable `$vod_notification_id`
 
+#### Closed Captions
+
+Mandatory fields:
+* `id` - a string that identifies the embedded captions. This will become the `INSTREAM-ID` field and must
+have one of the following values: `CC1`, `CC3`, `CC3`, `CC4`, or `SERVICEn`, where `n` is between 1 and 63.
+* `label` - a friendly string that indicates the language of the closed caption track.
+* `language` - a 3-letter (ISO-639-2) language code that indicates the language of the closed caption track.
```

Comment:
I kept this required because according to https://developer.apple.com/documentation/http_live_streaming/hls_authoring_specification_for_apple_devices the language attribute is required for type `CLOSED-CAPTIONS`